### PR TITLE
Update Integration Testing Sample App and add metrics path

### PR DESCRIPTION
**Pull request by KKelvinLo**
**What problem does this PR solve?**
This PR addresses [issue#24](https://github.com/aws-observability/aws-otel-js/issues/24) by adding a metrics path to the sample integration app and also resolves [issue#19](https://github.com/aws-observability/aws-otel-js/issues/19)

**Short description of Changes**
- added '/aws-sdk-call' to generate trace ID 
- added healthcheck path '/'
- added metrics path '/outgoing-http-call'
- removed client.js file (unused)
- updated workflow to run integration tests

### CODE REVIEW

File: integ-test/js-sdk-emitter/package.json [hunk 1]
```diff
@@ -17,21 +17,25 @@
...
-    "@opentelemetry/exporter-collector-grpc": "^0.12.0",
-    "@opentelemetry/node": "^0.11.0",
-    "@opentelemetry/plugin-http": "^0.11.0",
+    "@opentelemetry/api": "0.12.0",
+    "@opentelemetry/context-base": "^0.12.0",
+    "@opentelemetry/core": "^0.12.0",
+    "@opentelemetry/exporter-collector-grpc": "file:opentelemetry-js/packages/opentelemetry-exporter-collector-grpc",
+    "@opentelemetry/exporter-collector-proto": "file:opentelemetry-js/packages/opentelemetry-exporter-collector-proto",
+    "@opentelemetry/node": "^0.12.0",
+    "@opentelemetry/plugin-grpc": "^0.12.0",
```
Comment (anuraaga) [inline integ-test/js-sdk-emitter/package.json:26]: Do you use al these new plugins?

Comment (KKelvinLo) [reply integ-test/js-sdk-emitter/package.json:26]: No, I will clean it up, it's still a draft.

**Review by wyTrivail — APPROVED** [review]

File: sample-apps/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM node:14
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN ls
```
Comment (anuraaga) [inline sample-apps/Dockerfile:7]: Remove this, presumably for debugging

File: sample-apps/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+RUN cd sample-apps && npm install
```
Comment (anuraaga) [inline sample-apps/Dockerfile:9]: Use a single RUN command with lots of `&&` and `\` to break newlines, Docker images have their size determined by the number fo RUN commands, not just the final state.

File: sample-apps/metric-emitter.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+const { ConsoleLogger, LogLevel } = require('@opentelemetry/core');
+const { CollectorMetricExporter } = require('@opentelemetry/exporter-collector-grpc');
+const { MeterProvider } = require('@opentelemetry/metrics');
+
+const API_COUNTER_METRIC = 'apiBytesSent';
+const API_LATENCY_METRIC = 'latency';
+
+/** The OTLP Metrics gRPC Collector */
+const metricExporter = new CollectorMetricExporter({
+    serviceName: process.env.OTEL_RESOURCE_ATTRIBUTES ? process.env.OTEL_RESOURCE_ATTRIBUTES : 'aws-otel-integ-test',
```
Comment (anuraaga) [inline sample-apps/metric-emitter.js:12]: Should this be `OTEL_RESOURCE_ATTRIBUTES['sevice.name']`?

File: sample-apps/metric-emitter.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+function emitsPayloadMetric(bytes, apiName, statusCode) {
+    console.log('emit metric with http request size ' + bytes + ' byte, ' + apiName);
+    const labels = { 'apiName': apiName, 'statusCode': statusCode };
+    payloadMetric.bind(labels).add(bytes);
+}
+
+module.exports = {
+    emitReturnTimeMetric: emitReturnTimeMetric,
+    emitsPayloadMetric: emitsPayloadMetric
+}
```
Comment (anuraaga) [inline sample-apps/metric-emitter.js:59]: Newline

Comment (anuraaga) [reply sample-apps/metric-emitter.js:59]: Are we applying lint / prettier to the sample apps?

File: sample-apps/Dockerfile (hunk 3 above)
Comment (KKelvinLo) [reply sample-apps/Dockerfile:9]: Good point! I was not aware of that

File: sample-apps/metric-emitter.js (hunk 4 above)
Comment (KKelvinLo) [reply sample-apps/metric-emitter.js:12]: In the workflow I think OTEL_RESOURCE_ATTRIBUTES only has one value in the workflow, do I need to change that?

File: sample-apps/metric-emitter.js (hunk 5 above)
Comment (KKelvinLo) [reply sample-apps/metric-emitter.js]: Other than locally, no

File: sample-apps/metric-emitter.js (hunk 4 above)
Comment (anuraaga) [reply sample-apps/metric-emitter.js:12]: Yeah - the way to set that variable to reflect real-world use is `OTEL_RESOURCE_ATTRIBUTES=service.name=`

Comment (KKelvinLo) [reply sample-apps/metric-emitter.js]: done

**Review by anuraaga — APPROVED** [review]

**Pull request merged**